### DISCUSSION TITLE
no active connection

### DISCUSSION THREAD

**Discussion by ceccos83** [dbeaver/dbeaver#22994] [Q&A]
Hello, in the last few days with the last version and the previous one, sometimes when i run a query an error appears: "no active connection" if i click on SQL editor  "set active connection" it works but why does it lose connection?
thanks

**Comment by LonwoLonwo** [comment]
Hello @ceccos83 

What database do you use?

**Reply by ceccos83** [reply]
Vertica

**Reply by LonwoLonwo** [reply]
Do you have any default schema name for your Vertica connection?

![2024-03-01 19_16_26-Window](https://github.com/dbeaver/dbeaver/assets/45152336/de51cc40-57d2-46d9-b466-9e0f434a7506)

**Reply by ceccos83** [reply]
yes vertica is also the name of Database

**Reply by LonwoLonwo** [reply]
I can't reproduce this behavior, unfortunately.
Could you please check that a datasource metadata is available for your script? 

![2024-03-04 13_27_08-Window](https://github.com/dbeaver/dbeaver/assets/45152336/369e81ec-428c-4742-97cb-75a60e574c66)

**Reply by ceccos83** [reply]
i have this ![image (54)](https://github.com/dbeaver/dbeaver/assets/161628506/b5f80834-b802-4dee-b7ac-698ac5a748a9)
